eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,11 @@
+<!-- See aka.ms/dotnet/prebuilts for guidance on what pre-builts are and how to eliminate them. -->
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*" />
   </IgnorePatterns>
-</UsageData>
+  <Usages>
+    <!-- MSBuild versions starting from 16.10.0 do not target NET Core -->
+    <!-- and fail the non-source-build -->
+    <Usage Id="Microsoft.Build" Version="16.8.0" IsDirectDependency="true" />
```

Comment:
Should an SBRP be used here instead?

Reply:
SBRP has 16.10+ or 15.7 versions, 16.10 cannot be used per comment + did not want to downgrade a major version.

Can add the 16.8 or 16.9 back to SBRP and then remove this exclusion (wasn't sure, wanted to discuss this beforehand)

Reply:
Repo owners can comment on the specific version needed.  The exclusion here will need to be addressed otherwise this will be reported as a prebuilt in the product build once the per repo pvp feature is enabled because it will force the product build to use the version specified in this repo since the dependency is not marked as source-build.

Reply:
@tmat the version currently used is 16.7.0; does it makes sense from your perspective to keep using this version?

Reply:
We can use any msbuild version that works. The repo defines msbuild tasks, not using any special msbuild features.

Reply:
With changes in https://github.com/dotnet/sourcelink/pull/1001 we can use 17.3.2 which already has an SBRP for it, removed the exclusion